sdk/python/feast/dqm/errors.py

```diff
@@ -0,0 +1,13 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from .profilers.profiler import ValidationReport
+
+
+class ValidationFailed(Exception):
```

Comment:
nit: more specific like GEValidationFailed?

Reply:
Currently `Profile.validate` returns pretty generic `ValidationReport` in case of validation failure. So no need in specific exception (as of now)